pygeos/test/test_predicates.py

```diff
@@ -111,6 +111,38 @@ def test_relate_none(g1, g2):
     assert pygeos.relate(g1, g2) is None
 
 
+def test_relate_pattern():
+    line_string = pygeos.linestrings([(0, 0), (1, 0), (1, 1)])
+    polygon = pygeos.box(0, 0, 2, 2)
+    assert pygeos.relate(line_string, polygon) == "11F00F212"
+    assert pygeos.relate_pattern(line_string, polygon, "11F00F212")
+    assert pygeos.relate_pattern(line_string, polygon, "*********")
+    assert not pygeos.relate_pattern(line_string, polygon, "F********")
+
+
+@pytest.mark.parametrize("g1, g2", [(point, None), (None, point), (None, None)])
+def test_relate_pattern_none(g1, g2):
+    assert pygeos.relate_pattern(g1, g2, "*" * 9).item() is False
+
```

Comment:
Suggestion: add a test for `pattern=None`

Reply:
Also consider adding tests of empty geometries.

Reply:
Added some more tests

Reply:
I'm assuming that changes (if any) that come out of #253 will impact the tests of `None` vs `None` here as well (since they are hardcoded to `False` here); resolving those changes would be in a different PR (if any).